src/main/webapp/dev/js/common/remindModal.es6

```diff
@@ -9,19 +22,20 @@ function prepareRemindModal() {
             url: actionlink,
             beforeSend() {
                 $('#studentList').html('<img class="margin-center-horizontal" src="/images/ajax-loader.gif"/>');
-                $('#remindModal input[type="submit"]').prop('disabled', true).prop('value', 'Loading...');
+                $('#remindModal .remind-particular-button').prop('disabled', true).prop('value', 'Loading...');
             },
             error() {
                 $('#studentList').html('Error retrieving student list. Please close the dialog window and try again.');
             },
             success(data) {
                 setTimeout(() => {
                     $('#studentList').html(data);
-                    $('#remindModal input[type="submit"]').prop('disabled', false).prop('value', 'Remind');
+                    $('#remindModal .remind-particular-button').prop('disabled', false).prop('value', 'Remind');
                 }, 500);
             },
         });
     });
+    bindRemindButton();
```

Comment:
This does not follow SLAP. The routine above (`$('#remindModal').on('show.bs.modal', ...`) is shown in full and yet this one is extracted into a method that is only called once. Consider just putting the whole method here.